src/main/scala/com/softwaremill/realworld/GlobalLoggingDefectHandler.scala

```diff
@@ -0,0 +1,20 @@
+package com.softwaremill.realworld
+
+import utils.Exceptions
+import sttp.model.StatusCode
+import sttp.monad.MonadError
+import sttp.tapir.server.interceptor.exception.{ExceptionContext, ExceptionHandler}
+import sttp.tapir.server.model.ValuedEndpointOutput
+import sttp.tapir.{statusCode, stringBody}
+import zio.{Cause, ZIO}
+
+class GlobalLoggingDefectHandler[F[_]] extends ExceptionHandler[F]:
```

Comment:
I think the logging interceptor comes before the exception one, so the exceptions should be already logged?

Reply:
Plus I think we can create an exception handler for a specific `F` - here `RIO` or `Task`